cmds/live.c

```diff
@@ -203,12 +203,33 @@ static void setup_child_environ(struct uftrace_opts *opts)
 	free(libpath);
 }
 
+/**
+ * forward_option - forward a given option and its data to the agent
+ * @fd - agent socket file descriptor
+ * @opt - option to send
+ * @data - option data load
+ * @size - size of the option data load
+ * @return - status: < 0 on error, >= on success
+ */
+static int forward_option(int fd, enum agent_opt opt, void *value, size_t size)
```

Comment:
I wanted to centralize debug output. And later add a capability feature, to check if the agent supports a given feature. This would fit in this function.

Reply:
Ok, then it'd be fine.